spec/features/admin/orders_spec.rb

```diff
@@ -101,18 +101,18 @@ def new_order_with_distribution(distributor, order_cycle)
   scenario "displays error when incorrect distribution for products is chosen" do
     d = create(:distributor_enterprise)
     oc = create(:simple_order_cycle, distributors: [d])
-    puts d.name
-    puts @distributor.name
 
-    @order.state = 'cart'; @order.completed_at = nil; @order.save
+    # Moves the order back to the cart state
+    #   A nil user keeps the order in the cart state
+    #   Even if the edit page tries to automatically progress the order workflow
+    @order.state = 'cart'; @order.user = nil; @order.completed_at = nil; @order.save
```

Comment:
why not in multiple lines? we can even extract a method where we can document in detail why this is needed.

Reply:
:+1:
I keep it as multilines in the same place without extracting to method, I think the test is simple enough to take this.